admin_guide/backup_restore.adoc

```diff
@@ -0,0 +1,153 @@
+= Backup and Restore
+{product-author}
+{product-version}
+:data-uri:
+:icons: font
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+
+// REVIEWERS: READ THIS!
+//
+// In the following text, there are questions of the form:
+//   //??? QUESTION
+// Please feel free to make a line-comment to answer them, in addition
+// to any other (line-)comments on the correctness of the text.
+//
+// - Usually, the question pertains to the text preceding it.
+//   Questions pertaining to following text are explicitly noted.
+//
+// - There are bunch of questions at the end.
+//
+// Thanks for your cooperation on this (experimental) method of
+// refining the documentation.  Hopefully it will bear good fruit.
+
+== Overview
+
+In {product-title}, you can
+_back up_, saving state to separate storage,
+and _restore_, recreating state from separate storage,
+at the cluster level.
+There is also some preliminary support for
+xref:#project-backup[per-project backup].
+The full state of a cluster installation includes:
+
+- etcd data on each master
+- API objects
+- registry storage
+- volume storage
+
+This topic does not cover how to back up and restore
+link:../install_config/persistent_storage/index.html[persistent storage],
+as those topics are left to the underlying storage provider.
+
+
+[[cluster-backup]]
+== Cluster Backup
+
+. Save all the certificates and keys, on each master:
++
+----
+# cd /etc/origin/master
+# tar cf /tmp/certs-and-keys-$(hostname).tar \
+    master.proxy-client.crt \
+    master.proxy-client.key \
+    proxyca.crt \
+    proxyca.key \
+    master.server.crt \
+    master.server.key \
+    ca.crt \
+    ca.key \
+    master.etcd-client.crt \
+    master.etcd-client.key \
+    master.etcd-ca.crt
+----
+//??? What is missing?
+//??? What is unnecessary?
+
+. Create an *etcd* backup on each master:
++
+----
+# yum install etcd
+# etcdctl backup \
+    --data-dir /var/lib/origin/openshift.local.etcd \
+    --backup-dir /var/lib/origin/openshift.local.etcd.bak
+----
+
+. Create a template for all cluster API objects:
++
+====
+----
+$ oc export all \
+    --exact \//<1>
+    --all-namespaces \
+    --as-template=cluster.template
+----
+<1> Preserve fields that may be cluster specific,
+such as service `portalIP`s or generated names.
```

Comment:
The `s` directly following the markup messes up rendering at least in asciibinder builds and GH preview.